### ISSUE TITLE
Normalizing constant for q(z|ψ)

### ISSUE THREAD

**Issue by artsobolev** [mingzhang-yin/SIVI#1]
Hello

Thank you for posting the source code for your experiments! While viewing it I noticed that computation of the density q(z|ψ) at [SIVAE.py#L113](https://github.com/mingzhang-yin/SIVI/blob/2ae301519c33d35db41f8b0c83fac0c5b58fe18c/SIVAE.py#L113) seemingly lacks the normalizing constant that includes the changing variance term.

Am I missing something? Is it cancelled out somehow?

**Comment by mingzhang-yin** [comment]
Hi,

Thanks for your interest in SIVI. The normalizing constant with covariance is on #L114 log_H_iw = tf.log(tf.reduce_mean(ker,axis=2))-0.5*tf.reduce_sum(z_logv_iw,2)
